src/main/java/org/commcare/formplayer/application/FormController.java

```diff
@@ -193,6 +204,11 @@ private FormEntryResponseBean saveAnswer(AnswerQuestionRequestBean answerQuestio
         String fileId = null;
         Path mediaDirPath = formEntrySession.getMediaDirectoryPath(restoreFactory.getDomain(),
                 restoreFactory.getUsername(), restoreFactory.getAsUsername(), storageFactory.getAppId());
+
+        if (clear) {
```

Comment:
if `clear` is `@Nullable` this could cause a `NullPointerException`

Reply:
handled here https://github.com/dimagi/formplayer/pull/1459/commits/9af5cf36ff32d6e670942af67904d596f29d235e

Reply:
booleans doesn't need to `@Nullable`, they have a definite value of `true` or `false`

Reply:
ok, so we'll just keep the `if (cear)` check and take away the `@Nullable' annotation.

Reply:
@shubham1g5 `boolean` is `true` or `false` but `Boolean` can be `null`.

Reply:
that is right, think we can simply use the primitive `boolean` here. 

Even if we use `Boolean`, they can't be null in this context of code as we are always passing a non null value.  So it seems ok to remove the `@Nullable` annotation and not do the null handling.